lib/vm/src/trap/traphandlers.rs

```diff
@@ -385,12 +387,14 @@ static mut IS_WASM_PC: fn(usize) -> bool = |_| false;
 /// program counter is the pc of an actual wasm trap or not. This is then used
 /// to disambiguate faults that happen due to wasm and faults that happen due to
 /// bugs in Rust or elsewhere.
-pub fn init_traps(is_wasm_pc: fn(usize) -> bool) {
-    static INIT: Once = Once::new();
-    INIT.call_once(|| unsafe {
+pub fn init_traps(is_wasm_pc: fn(usize) -> bool, install_handlers: bool) {
+    unsafe {
         IS_WASM_PC = is_wasm_pc;
-        platform_init();
-    });
+        if !HANDLERS_INSTALLED && install_handlers {
+            platform_init();
+            HANDLERS_INSTALLED = true;
+        }
+    }
 }
```

Comment:
Rather than having this function receive an argument about installing handlers or not, it should be simply not be called. It should have the same arguments as before.

If the name is confusing, we can rename it to `init_trap_handlers`

Reply:
Done.